plugins/httpapi/zabbix.py

```diff
@@ -9,12 +9,23 @@
 
 DOCUMENTATION = """
 ---
-name: jsonrpc
-author: Markus Fischbacher (@rockaut)
+name: zabbix
+author:
+  - Markus Fischbacher (@rockaut)
+  - Evgeny Yurchenko (@BGmot)
 short_description: HttpApi Plugin for Zabbix
 description:
   - This HttpApi plugin provides methods to connect to Zabbix over their HTTP(S)-based api.
-version_added: 1.6.0
+version_added: 1.8.0
```

Comment:
I wonder if we do rather merge this to 2.0.0-dev branch, also `version_added:` is not the version collection is at, but what version it will be after the merge and next release. So in this case it would be 1.9.0

Reply:
I am sorry what version should I set it too here? 1.8.0 or 1.9.0? I don't know how release "thing" works and what it actually means -(.

Reply:
In regards do documentation - it's a big question for me. Ideally `ansible_network_os` and `ansible_connection` variables need to be described for the whole collection somewhere here https://docs.ansible.com/ansible/latest/collections/community/zabbix/index.html as it applies to all modules. Similar to what's done here for example https://docs.ansible.com/ansible/latest/network/user_guide/platform_nxos.html (though is it not a collection). User should not go to GitHub readme or docs/ to know how to correctly use the collection. So what do I do? (I am new to Ansible)

Reply:
> I am sorry what version should I set it too here? 1.8.0 or 1.9.0? I don't know how release "thing" works and what it actually means -(.

No problem. It actually is whatever we set there. Only guidelines we created are in the docs/ folder. Specifically this should state what is the first version on Ansible Galaxy that included this feature. Version is pushed there with each git tag on main.

I was thinking that since roles are being enhanced/cleaned up in a (sometimes) backwards incompatible way and prepared for the 2.0.0 release of this collection (huuge thanks to @pyrodie18 for that), we may also merge this to 2.0.0 as well. Otherwise next release will be 1.9.0.

Alternative is to merge this httpapi feature into next release 1.9.0 now and then drop whole zabbix-api for 2.0.0, but that would be on such a short notice we may not have a chance to catch any bugs.

Any opinions here?

Reply:
I don't know that I love taking away zabbix-api that fast.  I'd go ahead and release this now but if we can keep the current way working in a deprecated status that would be good and then get rid of it with 3.0.  Now we just gotta do a better job of getting 3.0 out a little bit sooner.

Reply:
Please take a look at the latest push. I tried to do something to resolve questions but I don't feel like it is enough. Still struggling with documentation part.